pywr/parameters/_activation_functions.pyx

```diff
@@ -80,7 +83,7 @@ cdef class RectifierParameter(Parameter):
         if self._value <= 0.0:
             return 0.0
         else:
-            return self.max_output * self._value / self._upper_bounds
+            return (self.max_output - self.min_output) * self._value / self._upper_bounds
```

Comment:
Tests are failing because this needs to be:

`return self.min_output + (self.max_output - self.min_output) * self._value / self._upper_bounds
`